public/app/plugins/datasource/cloudwatch/types.ts

```diff
@@ -332,6 +332,7 @@ export interface StartQueryRequest {
    * The list of log groups to be queried. You can include up to 20 log groups. A StartQuery operation must include a logGroupNames or a logGroupName parameter, but not both.
    */
   logGroupNames?: string[];
+  logGroups?: SelectableResourceValue[];
```

Comment:
Let's add a comment to this prop too. 

It's a bit hard to use the new [`logGroups`](getLogGroupFields) field from the `CloudWatchLogsQuery` since the type is not very explicit. I think SelectableResourceValue.value represents account id and SelectableResourceValue.label is account label but I might be wrong? Not for this PR, but we should probably change the typings here before removing the crossAccountQuerying feature toggle. Thoughts @sarahzinger?

Reply:
yeah @sunker i agree. I think when we do this task: "Migrate `log-groups` and `all-log-groups` resource request to new architecture" we can change the return type from the backend so that we're always returning a LogGroup rather than a "suggestData" and then it becomes easier on the frontend as well.

Reply:
Sounds good!